common/source/docs/common-future-wiki-changes.rst

```diff
@@ -16,16 +27,30 @@ on :ref:`common-powermodule-landingpage`, add:
     Synthetic Current Sensor/Analog Voltage Monitor <common-synthetic-current-monitor>
 
 
+on :ref:`common-uavcan-setup-advanced` , :ref:`mission-planner-initial-setup` , :ref:`common-slcan-f4`, and :ref:`common-slcan-f7h7` pages add the following note:
```

Comment:
```suggestion
on :ref:`common-uavcan-setup-advanced`, :ref:`mission-planner-initial-setup`, :ref:`common-slcan-f4`, and :ref:`common-slcan-f7h7` pages add the following note:
```

Off by one line due to the `[copywiki]` tag being removed during the build process.